Index: src/OneClickInstallWorkerFunctions.ycp
===================================================================
--- src/OneClickInstallWorkerFunctions.ycp	(revision 46084)
+++ src/OneClickInstallWorkerFunctions.ycp	(working copy)
@@ -141,8 +141,26 @@
 		foreach (string name, packages,
 		{
 			print(sformat(_("Marking package %1 for installation"),name));
+			if (!Pkg::PkgInstall(name))
-			//Prefer packages from repositories specified in the YMP
-			boolean inYmpRepos = false;
-			foreach (integer id, sourceids, 
 			{
-			 	y2debug("Looking for %1 in %2",name,id);
-			 	inYmpRepos = Pkg::ResolvfableInstallRepo(name,`package,id);
-				if (inYmpRepos)
-				{
-				 	y2debug("Found %1 in %2",name,id);
-				 	break;
-				} else
-				{
-				 	y2debug("Didn't find %1 in %2", name, id);
-				}
-			});
-			if (!inYmpRepos)
-			 	y2debug("Didn't find %1 At ALL in any YMP repos",name);
-			//If we didn't find it in the repos specified in the YMP try any repo.
-			if (!inYmpRepos && !Pkg::PkgInstall(name))
-			{
 				print(sformat(_("Warning: package %1 could not be installed."),name));
 				if(GUI)
 					OneClickInstallWorkerResponse::AddFailedPackage(name);
