Index: src/modules/OneClickInstallWidgets.rb
===================================================================
--- src/modules/OneClickInstallWidgets.rb.orig	2014-02-19 14:01:48.117821079 +0100
+++ src/modules/OneClickInstallWidgets.rb	2014-02-28 10:00:16.956465390 +0100
@@ -68,7 +68,7 @@
         Heading(_("Installation not possible")),
         RichText(
           _(
-            "The install link or file you opened does not contain instructions for this version of openSUSE."
+            "The install link or file you opened does not contain instructions for this version of SUSE Linux."
           )
         )
       )
